assignment1/meg-assignment.py

```diff
@@ -0,0 +1,19 @@
+class HumbleThings:
```

Comment:
I disagree that this is humble. This is looking a lot more complex and impressive than mine! Rename the class!